docs/docs/rest-api/public/api/v2/types/podStatus.raml

```diff
@@ -159,6 +163,10 @@ types:
       containers?:
         type: ContainerStatus[]
         description: status for each running container of this instance.
+      specReference?:
```

Comment:
we don't report the runSpecVersion as an instance-status field. we could. if we do, would you like that in this PR or a follow-up PR?